src/components/Summary.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment, ReactElement } from 'react'
+import { Fragment, ReactElement } from 'react'
```

Comment:
Not sure if you want to just do away with all `Fragment`'s at this point.

Reply:
Agree, will add